### DISCUSSION TITLE
This is a fantastic project, and will the author continue to update it?

### DISCUSSION THREAD

**Discussion by SuperStephenProMax** [zeit-ui/react#830] [General]
I really like the style of this project, but it seems like the creator @unix hasn't updated it in a long time. Next13 is coming out soon, will the author consider updating it for compatibility? Thank you very much for bringing such an excellent project. @unix

**Comment by AZProductions** [comment]
It would be amazing if someone could continue merging pr's and kinda keep the project alive, if unix isn't interested in maintaining it anymore. I'm more than happy to help out with the funding etc.

**Comment by unix** [comment]
Thank you to everyone who follows the project.

I'm starting a business these days, so it's hard to find time to review PR or write code for OSS, which is the reason why I haven't updated it in the past few months. I have to be honest and say that I don't have time to deal with anything in open source in the next few months either.

I'd love to see Geist get better with the help of others in the community. I've been looking for an experienced developer to take care of Geist, but no luck yet. Anyone with experience in open source projects who wants to take care of the maintenance of Geist, feel free to email me.

**Reply by robinhayward** [reply]
This is an amazing project, fingers crossed someone takes this on and thanks for all your work @unix. I looked at around at all the available UI libraries (never used one before) for a while, none of them captured me.. Geist did immediately and is incredibly easy to use.
